TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -9,6 +9,8 @@
     public static let yahooLauncher = TradeItYahooLauncher()
     internal static let linkedBrokerCache = TradeItLinkedBrokerCache()
 
+
+    internal static var _linkedBrokerManager: TradeItLinkedBrokerManager?
```

Comment:
Oh ok - can you move it down to above `public static var linkedBrokerManager: TradeItLinkedBrokerManager`